static/templates/admin_tab.handlebars

```diff
@@ -40,87 +40,111 @@
-              <th>Name</th>
-              <th>Email</th>
-              <th>Actions</th>
-            </tbody>
-          </table>
-          <div id="admin_page_users_loading_indicator"></div>
+            <div class="settings-section-title"><i class="icon-vector-user settings-section-icon"></i>
+            Users</div>
+            <table class="table table-condensed table-striped">
+              <tbody id="admin_users_table" class="admin_user_table">
+                <th>Name</th>
+                <th>Email</th>
+                <th>Actions</th>
+              </tbody>
+            </table>
+            <div id="admin_page_users_loading_indicator"></div>
           </div>
         </div>
         <div role="tabpanel" class="tab-pane" id="deactivated-users">
           <div id="admin-deactivated-users-list" class="settings-section">
-          <div class="settings-section-title"><i class="icon-vector-trash settings-section-icon"></i>
-          Deactivated Users</div>
-          <table class="table table-condensed table-striped">
-            <tbody id="admin_deactivated_users_table" class="admin_user_table">
-              <th>Name</th>
-              <th>Email</th>
-              <th>Actions</th>
-            </tbody>
-          </table>
-          <div id="admin_page_deactivated_users_loading_indicator"></div>
+            <div class="settings-section-title"><i class="icon-vector-trash settings-section-icon"></i>
+            Deactivated Users</div>
+            <table class="table table-condensed table-striped">
+              <tbody id="admin_deactivated_users_table" class="admin_user_table">
+                <th>Name</th>
+                <th>Email</th>
+                <th>Actions</th>
+              </tbody>
+            </table>
+            <div id="admin_page_deactivated_users_loading_indicator"></div>
           </div>
         </div>
         <div role="tabpanel" class="tab-pane" id="bots">
           <div id="admin-user-list" class="settings-section">
-          <div class="settings-section-title"><i class="icon-vector-github settings-section-icon"></i>
-          Bots</div>
-          <table class="table table-condensed table-striped">
-            <tbody id="admin_bots_table" class="admin_bot_table">
-              <th>Name</th>
-              <th>Email</th>
-              <th>Owner</th>
-              <th>Actions</th>
-            </tbody>
-          </table>
-          <div id="admin_page_bots_loading_indicator"></div>
+            <div class="settings-section-title"><i class="icon-vector-github settings-section-icon"></i>
+            Bots</div>
+            <table class="table table-condensed table-striped">
+              <tbody id="admin_bots_table" class="admin_bot_table">
+                <th>Name</th>
+                <th>Email</th>
+                <th>Owner</th>
+                <th>Actions</th>
+              </tbody>
+            </table>
+            <div id="admin_page_bots_loading_indicator"></div>
```

Comment:
Can you rebase so that you have the reindentation of the other sections of this page that aren't realm_emoji related as its own commit?  I'd like that to be separate from the new feature code, as per http://zulip.readthedocs.org/en/latest/code-style.html#version-control.